feature-detects/contextmenu.js

```diff
@@ -8,7 +8,7 @@
     "href": "https://www.w3.org/TR/html5/interactive-elements.html#context-menus"
   },{
     "name": "thewebrocks.com Demo",
-    "href": "https://thewebrocks.com/demos/context-menu/"
+    "href": "http://thewebrocks.com/demos/context-menu/"
```

Comment:
the only place where we have to go back to http? darn....

Reply:
Yes, for some reason http works but not https for this link.